deps/src/singular.cpp

```diff
@@ -53,6 +72,45 @@ JLCXX_MODULE define_julia_module(jlcxx::Module & Singular)
     singular_define_ideals(Singular);
     singular_define_matrices(Singular);
 
+    Singular.method("call_interpreter", [](std::string input) {
+        // set callbacks
```

Comment:
actually, shouldn't that be:
```suggestion
        // save callbacks
```
and then the next three lines have `// set callbacks`?